src/test/ProposalPayloadE2E.t.sol

```diff
@@ -3,37 +3,335 @@ pragma solidity ^0.8.15;
+            variableDebtToken: address(0), // Mock, no-validation because of the "dynamic" deployment on proposal exec
+            stableDebtToken: address(0), // Mock, no-validation because of the "dynamic" deployment on proposal execution
+            decimals: 18,
+            ltv: 2000,
+            liquidationThreshold: 4500,
+            liquidationBonus: 11000,
+            reserveFactor: 2000,
+            usageAsCollateralEnabled: true,
+            borrowingEnabled: true,
+            interestRateStrategy: NEW_INTEREST_RATE_STRATEGY_POLYGON,
+            stableBorrowRateEnabled: false,
+            isActive: true,
+            isFrozen: true
+        });
+
+        AaveV2Helpers._validateReserveConfig(expectedConfigPolygon, allConfigsPolygon);
+
+        AaveV2Helpers._validateInterestRateStrategy(
+            BAL_POLYGON,
+            ProposalPayloadPolygon(proposalPayloadPolygon).INTEREST_RATE_STRATEGY(),
+            InterestStrategyValuesV2({
+                excessUtilization: 20 * (AaveV2Helpers.RAY / 100),
+                optimalUtilization: 80 * (AaveV2Helpers.RAY / 100),
+                baseVariableBorrowRate: 3 * (AaveV2Helpers.RAY / 100),
+                stableRateSlope1: OLD_INTEREST_RATE_STRATEGY_POLYGON.stableRateSlope1(),
+                stableRateSlope2: OLD_INTEREST_RATE_STRATEGY_POLYGON.stableRateSlope2(),
+                variableRateSlope1: 14 * (AaveV2Helpers.RAY / 100),
+                variableRateSlope2: 150 * (AaveV2Helpers.RAY / 100)
+            }),
+            POLYGON
+        );
+
+        vm.selectFork(mainnetFork);
+        ReserveConfigV2[] memory allConfigsEthereum = AaveV2Helpers._getReservesConfigs(false, ETHEREUM);
+
+        ReserveConfigV2 memory expectedConfig = ReserveConfigV2({
+            symbol: "BAL",
+            underlying: BAL,
+            aToken: address(0), // Mock, no-validation because of the "dynamic" deployment on proposal execution
+            variableDebtToken: address(0), // Mock, no-validation because of the "dynamic" deployment on proposal exec
+            stableDebtToken: address(0), // Mock, no-validation because of the "dynamic" deployment on proposal execution
+            decimals: 18,
+            ltv: 6500,
+            liquidationThreshold: 7000,
+            liquidationBonus: 10800,
+            reserveFactor: 2000,
+            usageAsCollateralEnabled: true,
+            borrowingEnabled: false,
+            interestRateStrategy: NEW_INTEREST_RATE_STRATEGY_ETHEREUM,
+            stableBorrowRateEnabled: false,
+            isActive: true,
+            isFrozen: true
+        });
+
+        AaveV2Helpers._validateReserveConfig(expectedConfig, allConfigsEthereum);
 
+        AaveV2Helpers._validateInterestRateStrategy(
+            BAL,
+            ProposalPayload(proposalPayload).INTEREST_RATE_STRATEGY(),
+            InterestStrategyValuesV2({
+                excessUtilization: 20 * (AaveV2Helpers.RAY / 100),
```

Comment:
Just curious if things like `excessUtilization` should be changing from what the values are in the old strategy contract. If they aren't changing you should do something similar to what you've done for the stableRateSlope below (i.e `OLD_INTEREST_RATE_STRATEGY_ETHEREUM. excessUtilization()).

Same thing applies to all the other properties that are not meant to change for Ethereum V2, Polygon V2 and V3.

Reply:
@0xrajath the rest of the values are a function of the new interest rate strategy.
Excess utilization is 100 - the UOptimal for example.